src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -16315,6 +16274,72 @@ class A { public int Goo() { return 2; }}
             CreateCompilationWithMscorlib45(new[] { Parse(text, options: TestOptions.Script) }).VerifyDiagnostics(expectedDiagnostics);
         }
 
+        [Fact]
+        public void CS8036ERR_FieldInitializerInStruct()
```

Comment:
>CS8036ERR_FieldInitializerInStruct [](start = 20, length = 34)

Please move the test back to its previous location. I think it hasn't changed. Thanks

Reply:
Will do. Is there a reason for its strange location? I assumed it was better to place in numerically decreasing order, like every other test in the file.